packages/manager/apps/dedicated/client/app/otrs/otrs.routes.js

```diff
@@ -58,7 +58,7 @@ export default /* @ngInject */ ($stateProvider) => {
       breadcrumb: /* @ngInject */ ($translate) =>
         $translate.instant('otrs_title'),
       archived: /* @ngInject */ ($transition$) =>
-        $transition$.params().archived,
+        !!$transition$.params().archived,
```

Comment:
if duable easly, could be better like this `isArchived` ?

Reply:
We need to pass it to API, so maintaining the same name as expected by API as well.